packages/@ember/-internals/container/tests/container_test.js

```diff
@@ -666,30 +666,6 @@ moduleFor(
       assert.equal(instrance.ajax, 'fetch');
     }
 
-    ['@test #factoryFor does not add properties to the object being instantiated when _initFactory is present'](
```

Comment:
There is a skipped test that does this. One thing to note is that it will not work in IE11, since we don't have native symbol. We could update it to run if HAS_NATIVE_SYMBOL? Or we could `defineProperty` in IE11

Reply:
Which properties are added when native symbol isn't present?

Reply:
if HAS_NATIVE_SYMBOL is present, then only `LEGACY_OWNER` would be added (and that will be removed soon)

if HAS_NATIVE_SYMBOL is _not_ present, then `OWNER`, `LEGACY_OWNER`, and `INIT_FACTORY` would be added.

Reply:
Clarified that these symbols would only be added for _non-EmberObject_ based classes, where users manually created and registered their own classes.

Reply:
I think we chatted about this in DM's and decided that the _real_ thing we care about is that `Object.keys(EmberObject.create())` is empty, but that test will live in another spot.

@pzuraq - Is that added yet?